2.6 (#206)
nf-core/sarek: docs/variant_calling.md

# Variant calling <!-- omit in toc -->

- [Germline variant calling](#germline-variant-calling)
- [Somatic variant calling with tumor - normal pairs](#somatic-variant-calling-with-tumor---normal-pairs)
- [Somatic variant calling with tumor only samples](#somatic-variant-calling-with-tumor-only-samples)

## Germline variant calling

> Using Sarek, germline variant calling will be performed always, if a variant calling tool is selected and it allows for germline variant calling.

Reviewer: ```suggestion
Using Sarek, germline variant calling will always be performed if a variant calling tool is selected, and it germline variant calling is turned on.
```

Author: No sure about the phrasing for the last part.
```suggestion
Using Sarek, germline variant calling will always be performed if a variant calling tool with a germline mode is selected.
```
